### ISSUE TITLE
getFixedT keyPrefix with TS

### ISSUE THREAD

**Issue by riboher** [i18next/i18next#1856]
## 🐛 Bug Report

When using `getFixedT` function and passing a `keyPrefix` parameter, the resulting `t: TFunction` is not using the previous `keyPrefix` as intended and returns the whole translation template string.

## To Reproduce

es.json
```json
"layout": {
  "navbar": {
    "top": "someTranslation",
    "left": "otherTranslation"
  }
}
```
Valid
```js
const top = i18next.t('layout.navbar.top');
const left = i18next.t('layout.navbar.left');
```

Not valid
```js
const t = i18next.getFixedT(null, null, 'layout.navbar');
const top = t('top');
const left = t('left');
```

## Expected behavior

Both examples should be valid (and were prior to latest release)

## Your Environment

- *runtime version*: react 18
- *i18next version*: 22.0.2
- *react-i18next version*: 12.0.0
- *os*: Mac

**Comment by pedrodurek** [comment]
Thanks @riboher!
I missed the `getFixedT` function, I'll put a PR up to fix that

**Comment by riboher** [comment]
No worries @pedrodurek! Happy to help! 😃

**Comment by adrai** [comment]
should be fixed with v22.0.3
